Thermostat and Door Lock support

smart opened the pull request:
Opening this, followed the patterns and API docs on https://sdkcon78221.crestron.com/sdk/Crestron-Home-API/Content/Topics/API-Reference/Thermostats-API.htm

Tests were passing locally doing some device testing now and will confirm if works, would be easy to fill out the whole api from here. Thanks for the project.

smart committed:
thermostat support

smart committed:
doorlock support

smart committed:
bump

smart committed:
1.1.10

smart committed:
lint fixes

evgolsh commented:
Many thanks, looks good. Can you please also update the README.md and I'll merge then?

smart committed:
version bump

smart committed:
original package

smart commented:
Updated readme also added security system support. Im running my own npm package (homebridge-crestron-home-smart) of it and working well for me, will cut over to this after merged.

evgolsh commented:
Great work, thanks! I like the discovery logs and error handling. I published the npm 1.1.10. No need to run a fork, let's maintain this one.

smart commented:
Love it, yeah, no desire to maintain a fork, I just got really confused how
to run a dev package on my HA machine.

On Mon, Sep 8, 2025 at 6:16 PM Eugene Olshenbaum ***@***.***>
wrote:

> *evgolsh* left a comment (evgolsh/homebridge-crestron-home#19)
> <https://github.com/evgolsh/homebridge-crestron-home/pull/19#issuecomment-3268200072>
>
> Great work, thanks! I like the discovery logs and error handling. I
> published the npm 1.1.10. No need to run a fork, let's maintain this one.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/evgolsh/homebridge-crestron-home/pull/19#issuecomment-3268200072>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAADBTHV7USO4YFAL2BQPGT3RX52FAVCNFSM6AAAAACFHIWT3CVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTENRYGIYDAMBXGI>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Steve Martocci
@smart - http://www.twitter.com/smart
